gameoflife.test/TestGameOfLifeLibrary.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using Xunit;
+using gameoflife.lib;
+
+/*
+    Sources for making test as they should be:
+    - https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-dotnet-test
+    - https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices
+*/
+
+namespace gameoflife.test {
+
+    public class UnitTestGOLLibrary {
+        private readonly string correctGeneration1input = "Generation 1:\n4 8\n........\n....*...\n...**...\n........\n";
```

Comment:
these strings a butt ugly.
would write them as
```suggestion
        private readonly string correctGeneration1input = ""+
            "Generation 1:\n"+
            "4 8\n"+
            "........\n"+
            "....*...\n"+
            "...**...\n"+
            "........\n";
```

That or use a multiline string with `@""`

Reply:
also these strings should probably not be global data, they belong to individual tests.